src/category_theory/abelian/exact.lean

```diff
@@ -105,6 +108,18 @@ begin
   tfae_finish
 end
 
+lemma is_equivalence.exact_iff {D : Type u₁} [category.{v₁} D] [abelian D]
+  (F : C ⥤ D) [is_equivalence F] :
+  exact (F.map f) (F.map g) ↔ exact f g :=
+begin
+  haveI : preserves_limit (parallel_pair g 0) F := sorry,
+  haveI : preserves_colimit (parallel_pair f 0) F := sorry,
```

Comment:
These should follow from a suitable import. Probably
```
import category_theory.limits.preserves.shapes.kernels
```

Reply:
unfortunately it doesn't.

Reply:
In `category_theory.limits.preserves.shapes.kernels`, the relevant result is
```lean
instance : is_iso (kernel_comparison f G) :=
begin
  rw ← preserves_kernel.iso_hom,
  apply_instance
end
```
which depends on `variables [preserves_limit (parallel_pair f 0) G]`